include/zephyr/dt-bindings/pinctrl/rpi-pico-pinctrl-common.h

```diff
@@ -239,4 +239,35 @@
 #define USB_VBUS_DET_P25 RP2XXX_PINMUX(25, RP2_PINCTRL_GPIO_FUNC_USB)
 #define USB_VBUS_DET_P28 RP2XXX_PINMUX(28, RP2_PINCTRL_GPIO_FUNC_USB)
 
+#define SIO_P0  RP2XXX_PINMUX(0, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P1  RP2XXX_PINMUX(1, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P2  RP2XXX_PINMUX(2, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P3  RP2XXX_PINMUX(3, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P4  RP2XXX_PINMUX(4, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P5  RP2XXX_PINMUX(5, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P6  RP2XXX_PINMUX(6, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P7  RP2XXX_PINMUX(7, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P8  RP2XXX_PINMUX(8, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P9  RP2XXX_PINMUX(9, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P10 RP2XXX_PINMUX(10, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P11 RP2XXX_PINMUX(11, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P12 RP2XXX_PINMUX(12, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P13 RP2XXX_PINMUX(13, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P14 RP2XXX_PINMUX(14, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P15 RP2XXX_PINMUX(15, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P16 RP2XXX_PINMUX(16, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P17 RP2XXX_PINMUX(17, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P18 RP2XXX_PINMUX(18, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P19 RP2XXX_PINMUX(19, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P20 RP2XXX_PINMUX(20, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P21 RP2XXX_PINMUX(21, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P22 RP2XXX_PINMUX(22, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P23 RP2XXX_PINMUX(23, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P24 RP2XXX_PINMUX(24, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P25 RP2XXX_PINMUX(25, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P26 RP2XXX_PINMUX(26, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P27 RP2XXX_PINMUX(27, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P28 RP2XXX_PINMUX(28, RP2_PINCTRL_GPIO_FUNC_SIO)
+#define SIO_P29 RP2XXX_PINMUX(29, RP2_PINCTRL_GPIO_FUNC_SIO)
```

Comment:
There's nothing wrong with this fix itself.
However, there are still some things that need to be done to make it perfect.

https://github.com/zephyrproject-rtos/zephyr/blob/6c569c5bface15b92ed46cf42e6fc5ee589cf2c9/include/zephyr/dt-bindings/pinctrl/rpi-pico-rp2350-pinctrl-common.h#L70-L71